compiler/passes/ResolveScope.cpp

```diff
@@ -722,7 +795,8 @@ Symbol* ResolveScope::lookupForImport(Expr* expr, bool isUse) const {
       // certain this is okay to return
       retval = toSymbol(relativeScope->mAstRef);
       INT_ASSERT(retval != NULL);
-      return retval;
+      SymAndReferencedName res(retval, astr(""));
```

Comment:
Let's add `astr("")` to the "popular astrs" in symbol.h / symbol.cpp e.g. as `astrEmptyString` and use that here instead.